xbmc/platform/darwin/ios/XBMCController.mm

```diff
@@ -30,7 +32,13 @@
 #include "utils/URIUtils.h"
 #include "utils/log.h"
 #include "windowing/XBMC_events.h"
-#include "windowing/ios/WinSystemIOS.h"
+
+#import "platform/darwin/NSLogDebugHelpers.h"
+#import "platform/darwin/ios/IOSEAGLView.h"
+#import "platform/darwin/ios/IOSScreenManager.h"
+#import "platform/darwin/ios/XBMCApplication.h"
+#import "platform/darwin/ios-common/DarwinEmbedNowPlayingInfoManager.h"
+#import "windowing/ios/WinSystemIOS.h"
```

Comment:
wrong include order, https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#7-headers

Reply:
this is where ive got a question, windowing/ios/* is platform specific. Would this not be correct in that sense then?

Reply:
No, only files in `platform/*/` are currently flagged as platform specific, by calng-format.

Reply:
I wont argue, but id suggest maybe the guidelines be updated to be more specific then if there are these edge cases